include/libdnf5-cli/output/provides.hpp

```diff
@@ -31,7 +31,7 @@ namespace libdnf5::cli::output {
 
 enum ProvidesMatchedBy : int { NO_MATCH = 0, PROVIDES = 1, FILENAME = 2, BINARY = 3 };
 
-static void add_line_into_provides_table(struct libscols_table * table, const char * key, const char * value) {
+static inline void add_line_into_provides_table(struct libscols_table * table, const char * key, const char * value) {
```

Comment:
short answer, I didn't think about it since many of `add_line_...` functions are `static void` so I treated it as boilerplate, but let's think about it.

I believe here static would find use to limit the usage of the function in the header file only. I am now thinking if, to achieve the same goal, only the keyword `inline` could be used.

Reply:
I would recommend to remove `static` and `inline` from the function. Move code outside of header file (to `cpp`) and also remove `static` from templates bellow. I was unable to figure out any benefit with such a constructs here.

Reply:
I agree with this simplification, however, almost all the output functions would suffer from the same problem and the rework to keep the code consistent will be significant.

Reply:
Be honest I don't think that my suggestion will resolve the issue.

Reply:
I believe that the issue is not in project code but in applying `ccache` that is not compatible with the PR branch. This I experienced when locally I switched between significantly different branches. The error that it provided was always misleading and the fastest solution was to clean the cache and start from the beginning.